poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
+                        {{/if}}
+                    </label>
+                </div>
+            {{/if}}
+            <div class="percentage-gauge-container">
+              <div class="percentage-gauge" style="width:{{percent}}%;">
+                  <label class="poll-answer-label" for="answer-{{key}}">{{{answer}}}</label>
```

Comment:
@cptvitamin I don't want to write invalid HTML either, but as pointed out above, I don't see how to fix this without breaking backwards compatibility.  I've added a hint pointing out that you should only use bold or italics here.